src/WebJobs.Script/Rpc/Configuration/LanguageWorkerConfigOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public class LanguageWorkerConfigOptions
```

Comment:
I'd rename this LanguageWorkerOptions to make it more extensible in the future. It's not just about WorkerConfigs, and ConfigOptions is kinda redundant :) Then rename the below property to just WorkerConfigs.